client/Login.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import Link from '@mui/material/Link';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+
+
+const theme = createTheme();
+
+export default function Login() {
+  const [inputs, setInputs] = useState({
+    email: '',
```

Comment:
Use username here or allow login with email or username

Reply:
We agreed to eliminate username for login and from the user table of the database.

Reply:
ok sounds good